factcast-site/documentation-docsy/content/en/Setup/metrics.md

```diff
@@ -40,70 +40,68 @@ Furthermore, metrics in operations are automatically tagged with
+| subscribe-catchup          | `timer` | Time to create and return a catchup subscription (not the actual stream of facts).<br />Ref: [concepts](/concept)                                                                                                                                                                                                                                                                                                                                                 |
+| fetchById                  | `timer` | Time to get a fact from a given ID.                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| serialOf                   | `timer` | Time to get the serial of a fact.                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| enumerateNamespaces        | `timer` | Time to process namespaces enumeration.                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| enumerateTypes             | `timer` | Time to process types enumeration.                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| getStateFor                | `timer` | Time to get the latest state token for a given fact specification. The state contains the serial of the last fact matching the specifications, and is used by the client to determine whether a fact stream has been updated at a given point in time. Relevant for optimistic locking.<br />Ref: [optimistic locking](/usage/factus/optimistic-locking)                                                                                                          |
```

Comment:
s/contains/represents

it does not contain the serial, rather than *represent* the serial